source/common/http/http3_status_tracker_impl.cc

```diff
@@ -0,0 +1,126 @@
+#include "source/common/http/http3_status_tracker_impl.h"
```

Comment:
Is this file (and the .h) git mv'd from source/common/http/http3_status_tracker.cc? If so, man github's UI is so annoying!

Reply:
Right, github PR can't recognize `git mv` sometimes.

Reply:
Wow, after the diff gets reduced, github can recognize this `git mv` now.